serverless/event_workflows/serverless-kafka.adoc

```diff
@@ -1,6 +1,6 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-kafka"]
-= Using Apache Kafka with {ServerlessProductName}
+= Apache Kafka
```

Comment:
I don't like referencing upstream Apache here. It may leave the impression we support every Kafka installation.

Reply:
:man_shrugging: no opinion here

Reply:
Which kafka installations are supported?

Reply:
I think this came from Simon or someone originally re using the proper name but we can change it to Knative Kafka